design/craft/inclusivecache/src/QueuedRequest.scala

```diff
@@ -22,7 +22,7 @@ import chisel3._
 class QueuedRequest(params: InclusiveCacheParameters) extends InclusiveCacheBundle(params)
 {
   val prio   = Vec(3, Bool()) // A=001, B=010, C=100
-  val control= Bool() // control command
+  val control= Vec(2, Bool()) // control command (Flush=01, Invalidate=10)
```

Comment:
Can you make this a bundle
```
val control = new Bundle {
  val flush = Bool()
  val invalidate = Bool()
}
```

It would be logically equivalent, but simpler to comprehend

Reply:
Done. Thanks for the tip 👍